bugbug/model.py

```diff
@@ -499,8 +504,9 @@ def train(self, importance_cutoff=0.15, limit=None):
 
             tracking_metrics["report"] = report
 
+        # no confidence threshold - no need to handle  'Not classified' instances
```

Comment:
I think this comment is helpful in the context of reviewing this PR. However, without this context, the comment feels out of place (after merging the PR).